src/util/Constants.js

```diff
@@ -40,9 +45,11 @@ exports.DefaultOptions = {
   messageCacheMaxSize: 200,
   messageCacheLifetime: 0,
   messageSweepInterval: 0,
+  invalidRequestWarningInterval: 0,
   partials: [],
   restWsBridgeTimeout: 5000,
   restRequestTimeout: 15000,
+  restGlobalRateLimit: 0,
```

Comment:
why does this not default to 50?

Reply:
Two reasons:
- I thought it best to maintain the current behavior when I originally wrote this
- That would be a bad value to use for a bot with the mystery [higher global rate limit](https://discord.com/developers/docs/policy#api-limits) in >150k servers

I certainly don't object if the d.js team thinks it would be better to set a default of 50.